recipes/conllu/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "conllu" %}
+{% set version = "0.9" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "620033579ad690cddbb00524fea7d948b967c99301ee936580258441ff8509a4" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - conllu
+
+about:
+  home: https://github.com/EmilStenstrom/conllu/
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: CoNLL-U Parser parses a CoNLL-U formatted string into a nested python dictionary
+  description: CoNLL-U Parser parses a CoNLL-U formatted string into a nested python dictionary
+  doc_url: https://github.com/EmilStenstrom/conllu/
+  dev_url: https://github.com/EmilStenstrom/conllu/
+
+extra:
+  recipe-maintainers:
+    - nelson-liu
```

Comment:
@EmilStenstrom would you like to be listed as a recipe maintainer? Thanks for writing `conllu`.

Reply:
Hi @nelson-liu! Thanks for packaging conllu in conda, happy to see it used more! Since I don't use conda myself I don't think I should be a maintainer.